helix-core/src/main/java/org/apache/helix/controller/stages/AttributeName.java

```diff
@@ -38,5 +38,6 @@
   AsyncFIFOWorkerPool,
   PipelineType,
   LastRebalanceFinishTimeStamp,
-  ControllerDataProvider
+  ControllerDataProvider,
+  STATEFUL_REBALANCER
```

Comment:
Please comment if anyone of you has a better name. This one is not good.

Reply:
I think now, with the introduction of a StatefulRebalancer interface, having this name makes sense and pretty clear. What do you think?

Also, could we follow the enum convention? Use `StatefulRebalancer` instead?

Reply:
Thanks for the feedback.

As for the convention, it is not really clear to me. So I did some research. There are different opinions. The one that seems to be more reasonable to me is that enum items should be treated as constants, so they should follow constants name convention. Given this, uppercase letters make more sense here. I intend to enforce this as our enum convention unless a different rule has been documented somewhere else that I don't know : )

Reply:
Either way is fine - I realized there is no convention for enum.